smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftDependency.kt

```diff
@@ -49,204 +49,35 @@ class SwiftDependency(
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_READ_WRITE = SwiftDependency(
-            "SmithyReadWrite",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_TIMESTAMPS = SwiftDependency(
-            "SmithyTimestamps",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_XML = SwiftDependency(
-            "SmithyXML",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_JSON = SwiftDependency(
-            "SmithyJSON",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_FORM_URL = SwiftDependency(
-            "SmithyFormURL",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_WAITERS_API = SwiftDependency(
-            "SmithyWaitersAPI",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
+        val CLIENT_RUNTIME = smithySwiftTargetNamed("ClientRuntime")
+        val SMITHY = smithySwiftTargetNamed("Smithy")
+        val SMITHY_IDENTITY = smithySwiftTargetNamed("SmithyIdentity")
+        val SMITHY_RETRIES_API = smithySwiftTargetNamed("SmithyRetriesAPI")
+        val SMITHY_RETRIES = smithySwiftTargetNamed("SmithyRetries")
+        val SMITHY_HTTP_API = smithySwiftTargetNamed("SmithyHTTPAPI")
+        val SMITHY_HTTP_AUTH_API = smithySwiftTargetNamed("SmithyHTTPAuthAPI")
+        val SMITHY_STREAMS = smithySwiftTargetNamed("SmithyStreams")
+        val SMITHY_EVENT_STREAMS_API = smithySwiftTargetNamed("SmithyEventStreamsAPI")
+        val SMITHY_EVENT_STREAMS = smithySwiftTargetNamed("SmithyEventStreams")
+        val SMITHY_TEST_UTIL = smithySwiftTargetNamed("SmithyTestUtil")
+        val SMITHY_READ_WRITE = smithySwiftTargetNamed("SmithyReadWrite")
+        val SMITHY_TIMESTAMPS = smithySwiftTargetNamed("SmithyTimestamps")
+        val SMITHY_XML = smithySwiftTargetNamed("SmithyXML")
+        val SMITHY_JSON = smithySwiftTargetNamed("SmithyJSON")
+        val SMITHY_FORM_URL = smithySwiftTargetNamed("SmithyFormURL")
+        val SMITHY_WAITERS_API = smithySwiftTargetNamed("SmithyWaitersAPI")
+
+        private fun smithySwiftTargetNamed(name: String): SwiftDependency {
```

Comment:
This helper function makes the SwiftDependency declarations above a lot less repetitive.

Also, ignore the version numbers on Swift dependencies for now.  A future PR will automatically set them from a configuration file.

Reply:
All of these smithy-swift SwiftDependencies were also changed so that smithy-swift is a Git dependency, not installed from registry.